agot-bg-game-server/src/client/chat-client/ChatComponent.tsx

```diff
@@ -29,8 +28,8 @@ export default class ChatComponent extends Component<ChatComponentProps> {
         return (
             <div className="d-flex flex-column h-100">
                 <div className="flex-grow-1 mb-3 overflow-auto">
-                    {this.chatClient.channels.get(this.props.roomId).messages.slice().reverse().map(m => (
-                        <Row noGutters={true} className="flex-nowrap">
+                    {this.chatClient.channels.get(this.props.roomId).messages.slice().reverse().map((m, i) => (
```

Comment:
I think this is not a good `key` as the `key` of an entry in the chat will change since we `reverse` the list of messages. This modification might actually create UI inconsistencies since React will not re-render entries with the same key if they were already present.

Later, I will send the `id` of the message to the client, and this id will be used as a key.